ui_testing/testcases/header_tests/test08_usermanagement.py

```diff
@@ -85,78 +93,59 @@ def test004_download_user_sheet(self):
-        if this user not used in other entities
-        LIMS-6381
-        :return:
-            """
-        self.header_page.click_on_user_management_button()
+        self.base_selenium.click(element='header:user_management_button')
         # create new user
-        self.header_page.create_new_user(user_email=(self.header_page.generate_random_email()), user_role='',
-                                         user_password='1', user_confirm_password='1')
-        self.base_selenium.LOGGER.info('make sure that that the user record created in the active table')
-        active_user = self.header_page.search(value=self.header_page.user_name)[0].text
-        self.assertIn(self.header_page.user_name, active_user)
+        random_user_name = self.generate_random_string()
+        random_user_email = self.base_page.generate_random_email()
+        created_user =self.header_page.create_new_user(user_name=random_user_name,
+                                         user_email=random_user_email,
+                                         user_role='Admin', user_password='1',
+                                         user_confirm_password='1')
+        result = self.header_page.search(value=random_user_name)
+        self.assertTrue(result, created_user)
         self.header_page.select_all_records()
-        self.header_page.archive_selected_users()
-        self.header_page.get_archived_users()
+        self.header_page.archive_entity(menu_element='user_management:right_menu',
+                                        archive_element='user_management:archive')
+        self.header_page.get_archived_entities(menu_element='user_management:right_menu',
+                                               archived_element='user_management:archived')
         self.base_selenium.LOGGER.info('make sure that that the user record navigate to the archive table')
-        archive_user = self.header_page.search(value=self.header_page.user_name)[0].text
-        self.assertIn(self.header_page.user_name, archive_user)
+        result = self.header_page.search(value=random_user_name)
+        self.assertTrue(result, created_user)
         self.header_page.select_all_records()
-        self.header_page.click_on_user_right_menu()
-        self.header_page.click_on_delete_button()
+        self.base_selenium.LOGGER.info('Press on the right menu')
+        self.base_selenium.click(element='user_management:right_menu')
+        self.base_selenium.LOGGER.info('Press on the delete button')
+        self.base_selenium.click(element='user_management:delete')
         self.header_page.confirm_popup()
-        result = self.header_page.search(value=self.header_page.user_name)[0].text
+        result = self.header_page.search(value=random_user_name)
         self.base_selenium.LOGGER.info('deleted successfully')
-        self.assertFalse(result, 'deleted successfully')
+        self.assertTrue(result, 'No records found')
```

Comment:
- Log is wrong here should be after the assertion
- the result should be empty. No?

Reply:
Done